### PR TITLE
CLDSRV-492: fix monitoring for website head for 7.70

### PR DESCRIPTION
**Pull request by BourgoisMickael**
Some monitoring is present in 7.70 but not 7.10, so I didn't see it in integration branches in https://github.com/scality/cloudserver/pull/5510 when merging GET and HEAD files.

I matched version of websiteHead before the merge: https://github.com/scality/cloudserver/blob/b52f2356baf8709fceecff3dff2e72757375513a/lib/api/websiteHead.js

### CODE REVIEW

File: lib/api/website.js [hunk 1]
```diff
@@ -34,8 +34,10 @@ function _errorActions(err, errorDocument, routingRules,
         objectKey, err.code);
     if (errRoutingRule) {
         // route will redirect
-        monitoring.promMetrics(
-            'GET', bucketName, err.code, 'getObject');
+        if (request.method === 'GET') {
```
Comment (BourgoisMickael) [inline lib/api/website.js]: some monitoring is present in GET but not in HEAD for errors. I'm not sure if it's expected to be like that or if I should put the same monitoring in HEAD.

**Comment by BourgoisMickael** [comment]
ping

File: lib/api/website.js [hunk 2]
```diff
@@ -197,8 +200,10 @@ function website(request, log, callback) {
                 if (err) {
                     log.trace('error retrieving object metadata',
                     { error: err });
-                    monitoring.promMetrics(
-                        'GET', bucketName, err.code, 'getObject');
+                    if (request.method === 'GET') {
+                        monitoring.promMetrics(
```
Comment (BourgoisMickael) [inline lib/api/website.js]: this seems to duplicate monitoring for GET as there is also monitoring called in `_errorActions`

File: lib/api/website.js (hunk 1 above)
Comment (jonathan-gramain) [reply lib/api/website.js]: I think the same monitoring should be present for HEAD and GET, it's likely an overlook

File: lib/api/website.js [hunk 3]
```diff
@@ -136,6 +138,7 @@ function website(request, log, callback) {
         callback = callbackGetToHead(callback);
     }
     const methodCapitalized = capitalize(request.method);
+    const action = `${request.method.toLowerCase()}Object`;
```
Comment (jonathan-gramain) [inline lib/api/website.js]: I suggest using a regular `if (request.method === 'GET') {...}` etc., it's more readable and maintainable.

File: lib/api/website.js (hunk 2 above)
Comment (tmacro) [reply lib/api/website.js]: Looks duplicated to me as well. Probably should be removed

Comment (nicolas2bert) [reply lib/api/website.js]: Yes, there is monitoring duplication in the event of an error from metadata.getObjectMD. Could you please address this issue? Apart from that, everything looks good to me.

**Review by nicolas2bert — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**